### PR TITLE
Create 0000-01-02-Khushi-Shukla.md

### PR DESCRIPTION
**Pull request by CTRL-KHS**
Add Khushi-Shukla's file.

### CODE REVIEW

File: _posts/0000-01-02-Khushi-Shukla.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Khushi-Shukla) [inline _posts/0000-01-02-Khushi-Shukla.md:5]: If you have to shine like the sun, you have to burn like the sun

Comment (Khushi-Shukla) [inline _posts/0000-01-02-Khushi-Shukla.md:5]: If you have to shine like the sun, you have to burn like the sun.

Comment (Khushi-Shukla) [reply _posts/0000-01-02-Khushi-Shukla.md:5]: Rise and shine.

Comment (Khushi-Shukla) [reply _posts/0000-01-02-Khushi-Shukla.md:5]: hey

Comment (Khushi-Shukla) [inline _posts/0000-01-02-Khushi-Shukla.md:5]: "If you want to shine like the sun, you have to burn like the sun."

**Review by CTRL-KHS — COMMENTED** [review]
title: "If you want to shine like the sun, you have to burn like the sun"

File: _posts/0000-01-02-Khushi-Shukla.md (hunk 1 above)
Comment (Khushi-Shukla) [reply _posts/0000-01-02-Khushi-Shukla.md:5]: thank you.

**Review by CTRL-KHS — COMMENTED** [review]
great

File: _posts/0000-01-02-Khushi-Shukla.md (hunk 1 above)
Comment (Khushi-Shukla) [inline _posts/0000-01-02-Khushi-Shukla.md:5]: yes

**Review by CTRL-KHS — COMMENTED** [review]
approved

**Comment by Khushi-Shukla** [comment]
approving

**Pull request merged**